spec/library_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './lib/library'
+
+describe Library do
+  items = [{item:{ title: 'Alfons leker', author: 'A. Andersson' }, available: true, return_date: nil},
```

Comment:
Metrics/LineLength: Line is too long. [103/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)